strings/src/main/java/org/odk/collect/strings/format/LengthFormatter.kt

```diff
@@ -0,0 +1,18 @@
+package org.odk.collect.strings.format
+
+import java.util.Locale
+
+private const val ONE_HOUR = 3600000
+private const val ONE_MINUTE = 60000
+private const val ONE_SECOND = 1000
+
+fun formatLength(milliseconds: Long): String {
```

Comment:
Could totally see others disagreeing with moving this here. For me, it makes sense to have formatters etc live in `strings` but could see an argument that they should live in a separate utils module.

Reply:
Seems fine to me.